src/Models/BaseElement.php

```diff
@@ -1272,15 +1252,8 @@ public function EvenOdd()
      * @return string
      * @deprecated 5.3.0 Will be replaced with getTypeName()
      */
-    public static function getGraphQLTypeName(): string
+    public static function getTypeName(): string
```

Comment:
We now have `getType()`, `getTypeNice()`, and `getTypeName()` - can you please look and see if these can be joined together, and if not, see if they can be renamed to be more clearly distinct?

Reply:
There's this (kind of awful) concept of an Elemental "Type" which has many references to it, e.g. [ElementTypeRegistry](https://github.com/silverstripe/silverstripe-elemental/blob/5/src/Services/ElementTypeRegistry.php) and [elementTypeType](https://github.com/silverstripe/silverstripe-elemental/blob/5/client/src/types/elementTypeType.js) :-/ 

"TypeName" is basically elementType.name, or something like that

While I don't like this concept at all, it's also outside the scope of this PR to change this